core/net/ip/tcpip.c

```diff
@@ -155,6 +155,7 @@ unsigned char tcpip_is_forwarding; /* Forwarding right now? */
 PROCESS(tcpip_process, "TCP/IP stack");
 
 /*---------------------------------------------------------------------------*/
+#if UIP_TCP
```

Comment:
shouldn't this be `#if UIP_TCP || UIP_CONF_IP_FORWARD`?
(see https://github.com/feshie/contiki/blob/pr-warnings-fix/core/net/ip/tcpip.c#L180)